src/main/scala/gitbucket/core/util/Notifier.scala

```diff
@@ -19,8 +19,12 @@ import SystemSettingsService.Smtp
  * Please see the plugin for details.
  */
 trait Notifier {
+  def toNotify(subject: String, htmlMsg: String)
+    (recipients: Account => Session => Seq[String])(implicit context: Context): Unit = {
+    toNotify(subject, htmlMsg, None)(recipients)
+  }
 
-  def toNotify(subject: String, msg: String)
+  def toNotify(subject: String, htmlMsg: String, textMsg: Option[String])
```

Comment:
@kounoike thanks

> It doesn't break plugin interface.

As you say, it is supposed to be this way as it should be, but... we might as well add `textMsg: String` just this once if you fix the plugin right after this. How you do you feel about it?